src/RestEase/Implementation/SerializationMethods.cs

```diff
@@ -4,6 +4,7 @@ internal class ResolvedSerializationMethods
     {
         public const BodySerializationMethod DefaultBodySerializationMethod = BodySerializationMethod.Serialized;
         public QuerySerializationMethod DefaultQuerySerializationMethod = QuerySerializationMethod.ToString;
+        public PathSerializationMethod DefaultPathSerializationMethod = PathSerializationMethod.ToString;
```

Comment:
This and the `DefaultQuerySerializationMethod` should be `const`, I think

Reply:
Added const to both.